Sources/SwiftGodotMacroLibrary/MacroGodot.swift

```diff
@@ -137,9 +145,12 @@ class GodotMacroProcessor {
             guard let ips = singleVar.pattern.as(IdentifierPatternSyntax.self) else {
                 throw GodotMacroError.expectedIdentifier(singleVar)
             }
-            let varName = ips.identifier.text
-            let setterName = "_mproxy_set_\(varName)"
-            let getterName = "_mproxy_get_\(varName)"
+            let varNameWithPrefix = ips.identifier.text
+            let varNameWithoutPrefix = String(varNameWithPrefix.trimmingPrefix(prefix))
+            let proxySetterName = "_mproxy_set_\(varNameWithPrefix)"
+            let proxyGetterName = "_mproxy_get_\(varNameWithPrefix)"
+            let setterName = "set_\(varNameWithoutPrefix.camelCaseToSnakeCase())"
+            let getterName = "get_\(varNameWithoutPrefix.camelCaseToSnakeCase())"
```

Comment:
We want to make sure that the set/get functions in GDScript don't have the prefix in the function signature's name. So we must trim the prefix, and use that name.